front/src/routes/integration/all/netatmo/Dashboard-box-page/BinaryDeviceFeatureNetatmo.jsx

```diff
@@ -0,0 +1,76 @@
+import { Text, Localizer } from 'preact-i18n';
+import actions from './actions';
+import { connect } from 'unistore/preact';
+import { Component } from 'preact';
+
+import style from '../style.css';
+
+const RenderCommandGlobal = ({ children, ...props }) => {
+  function updateValue() {
+    props.updateValue(
+      props.x,
+      props.y,
+      props.device,
+      props.deviceFeature,
+      props.deviceIndex,
+      props.deviceFeatureIndex,
+      props.deviceFeature.last_value === 0 ? 1 : 0
+    );
+  }
+
+  return (
+    <Localizer>
+      <tr class={props.styleTest} title={props.title}>
+        <td>
+          <i class="fe fe-toggle-right" />
+        </td>
+        <td>{props.deviceFeature.name}</td>
+        <td class="text-right">
+          <label class="custom-switch">
+            <input
+              type="radio"
+              name={props.deviceFeature.id}
+              value="1"
+              class="custom-switch-input"
+              checked={props.deviceFeature.last_value}
+              onClick={updateValue}
+              disabled="disabled"
+            />
+            <Localizer>
+              <span class="custom-switch-indicator" title={<Text id="integration.netatmo.box.noCommand" />} />
+            </Localizer>
+          </label>
+        </td>
+      </tr>
+    </Localizer>
+  );
+};
+
+@connect('session,httpClient,user', actions)
+class BinaryDeviceType extends Component {
+  async getNetatmoConnect(netatmoIsConnect) {
+    netatmoIsConnect = await this.props.httpClient.get('/api/v1/service/netatmo/variable/NETATMO_IS_CONNECT');
```

Comment:
What is the NETATMO_IS_CONNECT variable about? I guess it's a boolean "netatmoIsConnected" ?

 I don't think this information should be stored in DB, doesn't make sense, this is pure live data, it should be RAM only

Reply:
Don't understand why it should be in ram ? Just a shutdown of the raspberry and you loose the information ? 

It's juste a simple way to be sure that Gladys is connected to Netatmo